src/components/Calculator.js

```diff
@@ -1,12 +1,18 @@
 import React from 'react';
 import '../App.css';
 import Buttons from './Buttons';
+import QuoteContainer from './QuoteContainer';
 
 function Calculator() {
   return (
-    <div className="calc">
-      <Buttons />
-    </div>
+    <body>
+      <h1 className="title">Math Magicians</h1>
+      <div className="calc">
+        <Buttons />
+        <QuoteContainer />
```

Comment:
- [ ] Kindly import the new component `QuoteContainer` into the main component `App` (next to the Calculator component) instead as per this [requirement](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/project_fetch_data_from_api.md#:~:text=Show%20an%20error-,state,the%20new%20component%20into%20your%20main%20component%20(next%20to%20the%20Calculator),-It%20should%20be).

Reply:
Ohhh boyy 😨😨😨

Yes, you're reason, Mr @gemmen29 I'll work on it 🫡